### PR TITLE
README: avoid ":latest" tag.

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

### CODE REVIEW

**Review by ricardoasmarques — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ configured and running on all machines in the cluster before triggering
 Finally we need to set the Ceph container image path:
 
 ```
-/containers/images/ceph set docker.io/ceph/daemon-base:latest
+/containers/images/ceph set docker.io/ceph/ceph:v15.2.2
```
Comment (smithfarm) [inline README.md:134]: How are users going to know which tag to use? (When installing and updating RPM packages, users are not required to specify a version number...)

When a SES7 maintenance update is released, will the users be expected to specify the exact version number of the maintenance update container image to update the system to it?

Comment (sebastian-philipp) [reply README.md:134]: that's the plan

Comment (sebastian-philipp) [reply README.md:134]: @asettle

**Pull request merged**